deploy/gke/delete-installed.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+helm list | awk 'NR>1 {print $1}' | xargs helm delete
+kubectl delete --all ing
+kubectl delete --all secret
+kubectl get cm | grep -v "kube" | awk 'NR>1 {print $1}' | xargs kubectl delete cm
+kubectl delete --all pvc
```

Comment:
Dùng loop để delete hết ing, secret, pvc, pv, .. đi sếp :v ai lại list hết như này

Reply:
khs dùng for loop ntn em thấy code còn khó đọc hơn sếp ạ:
```
resources=("ing" "secret" "cm" "pvc" "pv" "po" "job")

for resource in "${resources[@]}"; do
  if [ "$resource" == "cm" ]; then
    kubectl get cm | grep -v "kube" | awk 'NR>1 {print $1}' | xargs kubectl delete cm
  else
    kubectl delete --all "$resource"
  fi
done
```

Nhg mà em ghi nhận ý kiến =))